libc/time/strftime.c

```diff
@@ -31,28 +31,48 @@ struct test_data {
 	int ret;
 };
 
-static const struct tm t1 = { .tm_sec = 1, .tm_min = 1, .tm_hour = 6, .tm_mday = 2, .tm_mon = 2,
-							  .tm_year = 111, .tm_wday = 0, .tm_yday = 2, .tm_isdst = 0 };
-
-static const struct tm t2 = { .tm_sec = 11, .tm_min = 12, .tm_hour = 13, .tm_mday = 23, .tm_mon = 11,
-							  .tm_year = 95, .tm_wday = 6, .tm_yday = 235, .tm_isdst = 0 };
-
-static const struct tm t3 = { .tm_sec = 11, .tm_min = 12, .tm_hour = 13, .tm_mday = 23, .tm_mon = 11,
-							  .tm_year = 105, .tm_wday = 1, .tm_yday = 235, .tm_isdst = 0 };
+static const struct tm t1 = { .tm_sec = 1, .tm_min = 1, .tm_hour = 6, .tm_mday = 2, .tm_mon = 2, .tm_year = 111, .tm_wday = 0, .tm_yday = 2, .tm_isdst = 0 };
+static const struct tm t2 = { .tm_sec = 11, .tm_min = 12, .tm_hour = 13, .tm_mday = 23, .tm_mon = 11, .tm_year = 95, .tm_wday = 6, .tm_yday = 235, .tm_isdst = 0 };
+static const struct tm t3 = { .tm_sec = 11, .tm_min = 12, .tm_hour = 13, .tm_mday = 23, .tm_mon = 11, .tm_year = 105, .tm_wday = 1, .tm_yday = 235, .tm_isdst = 0 };
+static const struct tm t4 = { .tm_sec = 0, .tm_min = 0, .tm_hour = 0, .tm_mday = 2, .tm_mon = 0, .tm_year = 99, .tm_wday = 6, .tm_yday = 2, .tm_isdst = 0 };
 
-static const struct test_data test_vector[T1_LEN] = {
+static const struct test_data basic_formatting[] = {
```

Comment:
I think that we could place the declarations at the beginning of the file
like it's done in the rest of time tests. So sth like:
```suggestion
static const struct test_data basic_formatting[T1_LEN];
static const struct test_data additional_format_chars[T2_LEN];
...

static const struct test_data basic_formatting[] = {
```

Reply:
Then we could place test data below `TEST_GROUP_RUNNER`.

Reply:
Alternatively you could switch to the convention applied in this test in mktime/strftime tests - we just want to be consistent.